src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -664,7 +664,7 @@ private void logException(Exception exception, File file) {
                 Definitions.CHECKSTYLE_BUNDLE,
                 key,
                 args,
-                getId(),
+                null,
```

Comment:
This will mean that:
If used define id for module - https://checkstyle.org/config.html#Id
It will not be printed to user in report instead of module name, so just module name will be used. If user has two instances of same module type he will not know what module instance caused it.

Example of `id` printing instead  of module name in CLI output - https://github.com/checkstyle/checkstyle/issues/13038 

We can not cover such mutation by our Inputs with embedded config. But we should create test method that validate violation `id` with config defined in java code and assert on violation object field or checking content of some test audit listener https://checkstyle.org/config.html#Audit_Listeners we should already have some tests for this.

Reply:
ok i got it Thanks.

I found this type of testing at https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/AuditEventDefaultFormatterTest.java

i have update an exisiting test is it fine